dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/test/java/org/dashbuilder/renderer/c3/client/charts/meter/C3MeterDisplayerTest.java

```diff
@@ -93,4 +99,20 @@ public void c3meterColumnsExtractorsTest() throws ParseException {
         
     }
     
+    @Test
+    public void columnFormatTest() throws ParseException {
```

Comment:
added tests for column formatting, @ederign  - let me know if it is enough - thanks!